scripts/CEE/cluster-health-check/metadata.yaml

```diff
@@ -0,0 +1,95 @@
+file: run-oc-hc.sh
+name: cluster-health-check
+description: Check the overall health of an OpenShift cluster
+author: Givaldo Lins
+allowedGroups:
+  - CEE
+  - SREP
+  - CSSRE
+  - LPSRE
+  - MTSRE
+rbac:
+  clusterRoleRules:
```

Comment:
Do you need permissions to all namespaces?

Reply:
yes, because the check looks for things like failed pods and restrictive PDBs in all namespaces. Although customer workloads might not be a concern for SRE, that may cause disruption in their applications during an upgrade. It hasn't been implemented yet, but other things to check that would fall into that are pods using volumes from local nodes, node selection, single replica deployments, etc.